src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,111 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationServiceImpl;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final int DEFAULT_AGE = 27;
+    private static final int OTHER_AGE = 22;
+    private static final int WRONG_AGE1 = 0;
+    private static final int WRONG_AGE2 = -10;
+    private static final String DEFAULT_LOGIN = "login";
+    private static final String DEFAULT_PASSWORD = "password";
+    private static final String OTHER_PASSWORD = "another_Password";
+    private static final String WRONG_PASSWORD = "1234";
+    private static RegistrationServiceImpl regService;
```

Comment:
```suggestion
    private static RegistrationService registrationService;
```
Contractions  are a slippery slope)